nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/LockException.java

```diff
@@ -43,4 +43,9 @@ public LockException(Throwable cause) {
         super(cause);
     }
 
+    public LockException(String message, int statusCode) {
+        super(message);
+        this.statusCode = statusCode;
+    }
+
```

Comment:
Since you're adding this one, can you please add the other signatures defined in `NuxeoException` allowing to pass a statusCode?
```
public NuxeoException(int statusCode)
public NuxeoException(String message, Throwable cause, int statusCode)
public NuxeoException(Throwable cause, int statusCode)
```